playbook/app/pb_kits/playbook/pb_circle_chart/_circle_chart.jsx

```diff
@@ -43,6 +44,7 @@ const CircleChart = (props: CircleChartProps) => {
     dataLabelHtml = '<div>{point.name}</div>',
     dataLabels = false,
     headerFormat = null,
+    height = '400',
```

Comment:
@maricris-sn Can this unset by default? see https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_bar_graph/_bar_graph.jsx

Reply:
+1 ^^ Per highcharts docs, it's best to leave it unset. So you can just make this line say `height,`.

Reply:
@jasoncypret @jasperfurniss ah, right. yes, I can have this unset.  It's defaulting to 400 anyway as per highcharts.